libs/datamodel/core/tests/base/basic.rs

```diff
@@ -223,10 +222,16 @@ fn type_aliases_must_warn() {
       }
     "#};
 
-    let parsed = datamodel::parse_datamodel(dml).unwrap();
-    let expected = DatamodelWarning::DeprecatedTypeAlias {
-        span: Span { start: 0, end: 36 },
-    };
+    let error = datamodel::parse_schema(dml).map(drop).unwrap_err();
 
-    assert_eq!(Some(expected).as_ref(), parsed.warnings.first());
+    let expectation = expect![[r#"
+        [1;91merror[0m: [1mError validating: Invalid type definition. Please check the documentation in https://pris.ly/d/composite-types[0m
+          [1;94m-->[0m  [4mschema.prisma:1[0m
+        [1;94m   | [0m
+        [1;94m   | [0m
+        [1;94m 1 | [0m[1;91mtype MyString = String @default("B")[0m
+        [1;94m   | [0m
+    "#]];
```

Comment:
let's validate the error message with product if in doubt

Reply:
I'm not that much in doubt with this. Undocumented feature gets removed and we just give a hard validation error for now.

Reply:
+1 on treating these as they never happened (in this one, special case of the old, undocumented `type`)